2025-10-17 07:14:51,607 INFO LOB[ethusdt] - Initialized BinanceOrderBookStream
2025-10-17 07:14:51,607 INFO LOB[ethusdt] - REST snapshot URL: https://fapi.binance.com/fapi/v1/depth?symbol=ETHUSDT&limit=1000
2025-10-17 07:14:51,607 INFO LOB[ethusdt] - WebSocket URL:     wss://fstream.binance.com/stream?streams=ethusdt@depth@100ms
2025-10-17 07:14:51,608 INFO LOB[ethusdt] - Log file: C:\Users\user\Desktop\ofi_cvd_framework\ofi_cvd_framework\v13_ofi_ai_system\logs\ethusdt_20251016.log
2025-10-17 07:14:51,608 INFO LOB[ethusdt] - ============================================================
2025-10-17 07:14:51,608 INFO LOB[ethusdt] - Start Binance WebSocket client
2025-10-17 07:14:51,608 INFO LOB[ethusdt] - Symbol: ETHUSDT depth=5 url=wss://fstream.binance.com/stream?streams=ethusdt@depth@100ms
2025-10-17 07:14:51,608 INFO LOB[ethusdt] - Auto reconnect: on
2025-10-17 07:14:51,608 INFO LOB[ethusdt] - ============================================================
2025-10-17 07:15:12,680 ERROR LOB[ethusdt] - WebSocket error: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Programs\Python\Python311\Lib\site-packages\websocket\_app.py", line 484, in setSock
    self.sock.connect(
  File "C:\Users\user\AppData\Local\Programs\Python\Python311\Lib\site-packages\websocket\_core.py", line 256, in connect
    self.sock, addrs = connect(
                       ^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python311\Lib\site-packages\websocket\_http.py", line 145, in connect
    sock = _open_socket(addrinfo_list, options.sockopt, options.timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python311\Lib\site-packages\websocket\_http.py", line 222, in _open_socket
    raise error
  File "C:\Users\user\AppData\Local\Programs\Python\Python311\Lib\site-packages\websocket\_http.py", line 209, in _open_socket
    sock.connect(address)
TimeoutError: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
2025-10-17 07:15:12,680 WARNING LOB[ethusdt] - WebSocket closed
